website/core/Footer.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+class Footer extends React.Component {
+  docUrl(doc, language) {
+    const baseUrl = this.props.config.baseUrl;
+    const docsUrl = this.props.config.docsUrl;
+    const docsPart = `${docsUrl ? `${docsUrl}/` : ''}`;
+    const langPart = `${language ? `${language}/` : ''}`;
+    return `${baseUrl}${docsPart}${langPart}${doc}`;
+  }
+
+  pageUrl(doc, language) {
+    const baseUrl = this.props.config.baseUrl;
+    return baseUrl + (language ? `${language}/` : '') + doc;
+  }
+
+  render() {
+    return (
+      <footer className="nav-footer" id="footer">
+        <section className="sitemap">
+          <a href={this.props.config.baseUrl} className="nav-home">
+            {this.props.config.footerIcon && (
+              <img
+                src={this.props.config.baseUrl + this.props.config.footerIcon}
+                alt={this.props.config.title}
+                width="66"
+                height="58"
+              />
+            )}
+          </a>
+          <div>
+            <h5>Facebook Dev Circles Lahore</h5>
+            <a href={this.docUrl('doc1.html', this.props.language)}>
+              Getting Started (or other categories)
+            </a>
+            <a href={this.docUrl('doc2.html', this.props.language)}>
+              Guides (or other categories)
+            </a>
+            <a href={this.docUrl('doc3.html', this.props.language)}>
+              API Reference (or other categories)
+            </a>
+          </div>
+          <div>
+            <h5>Community</h5>
+            <a href={this.pageUrl('users.html', this.props.language)}>
+              User Showcase
+            </a>
+            <a
+              href="https://stackoverflow.com/questions/tagged/"
+              target="_blank"
+              rel="noreferrer noopener">
+              Stack Overflow
+            </a>
+            <a href="https://discordapp.com/">Project Chat</a>
+            <a
+              href="https://twitter.com/"
+              target="_blank"
+              rel="noreferrer noopener">
+              Twitter
+            </a>
```

Comment:
I believe we won't be needing Twitter since there isn't any twitter account for DevC, am I right? @mikaalanwar @Taimoormk